### DISCUSSION TITLE
can we define custom commands and bind them (e.g. switch-theme)?

### DISCUSSION THREAD

**Discussion by lost22git** [qutebrowser/qutebrowser#8040] [General] [answered]
as title

**Comment by The-Compiler** [comment, marked as answer]
You can chain commands with `;;` and use the `aliases` setting to define a new command based on that (or bind it to a key directly).

**Reply by lost22git** [reply]
Thank you for your reply, I also want to know if we can make the function as a command？

**Reply by The-Compiler** [reply]
Not officially yet, see #30. You can in theory import qutebrowser internals in a `config.py` and register a command, but that's unsupported, and you'll be on your own with that.
